Lib/asyncio/tools.py

```diff
@@ -246,6 +271,19 @@ def display_awaited_by_tasks_table(pid: int) -> None:
         print(f"{row[0]:<10} {row[1]:<20} {row[2]:<20} {row[3]:<50} {row[4]:<50} {row[5]:<15} {row[6]:<15}")
 
 
+def _display_awaited_by_tasks_csv(table, format: TaskTableOutputFormat) -> None:
+    match format:
+        case TaskTableOutputFormat.csv:
+            delimiter = ','
+        case TaskTableOutputFormat.bsv:
+            delimiter = '\N{BANANA}'
+        case _:
+            raise ValueError(f"Unknown output format: {format}")
+    csv_writer = csv.writer(sys.stdout, delimiter=delimiter)
+    csv_writer.writerow(_header)
```

Comment:
Can you:

- either name this variable `_CSV_HEADERS`
- or inline them.

Note that I preferred using the `_HEADERS[0]` as you did because if they change we'll need to change at two places as well...

Reply:
Yeah I don't like it either that the column names are repeated in two places but consensus seemed to steer in the other direction

Reply:
```suggestion
    csv_writer.writerow(csv_headers)
```